### ISSUE TITLE
Tests not building on Android Studio Version 4.0

### ISSUE THREAD

**Issue by miaboloix** [oppia/oppia-android#1238]
When building the app on Android Studio 4.0, I get the following error:

```
The option 'android.enableUnitTestBinaryResources' is deprecated.
The current default is 'false'.
It has been removed from the current version of the Android Gradle plugin.
The raw resource for unit test functionality is removed.
Affected Modules: app
```

**Comment by BenHenning** [comment]
@miaboloix can you add a bit more context in opening comment as to what wasn't working? E.g. what repro steps did you take to hit this issue, and what error did you see?

**Comment by anandwana001** [comment]
Is it something like this? @sajalasati He got this error  
![Screenshot from 2020-06-03 12-25-10](https://user-images.githubusercontent.com/25057618/83605893-57817200-a596-11ea-9bf8-ec75edb27aa8.png)

**Comment by miaboloix** [comment]
@BenHenning Is this okay for context (I edited the description)? I am struggling a bit with how to elaborate more than this. 
@anandwana001 Yes! That is exactly the error I got.

**Comment by sajalasati** [comment]
@miaboloix @BenHenning I could finally resolve the above error without commenting/deleting the line **android.enableUnitTestBinaryResources = true** in `gradle.properties`. I am working with Android Studio **version 4.0**. By mistake I had updated the below mentioned plugins as recommended by Android Studio during the very first Gradle Sync. When I reverted back the statements to original values as shown below:
- in the `build.gradle` file: from **classpath 'com.android.tools.build:gradle:4.0.0'** back to **classpath 'com.android.tools.build:gradle:3.6.1'**
- in the `/gradle/wrapper/gradle-wrapper.properties` file: from **distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip** back to **distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-all.zip**

i.e as mentioned exactly in the Oppia's original code too, everything started working fine - i.e without making any changes to `gradle.properties` file. These updated versions might/might not be the exact reason, but doing just these above mentioned changes worked for me. So in my opinion deleting the line **android.enableUnitTestBinaryResources = true** from `gradle.properties` is not needed if we go by the original code from the repo.
